components/centraldashboard/public/components/main-page.js

```diff
@@ -566,6 +580,55 @@ export class MainPage extends mixinBehaviors([AppLocalizeBehavior], utilitiesMix
         this.menuLinks = JSON.parse(JSON.stringify(this.menuLinks));
     }
 
+    /**
+     * Handles the list Notebooks Servers response to set date format and icon.
+     * @param {Event} responseEvent
+     */
+    async _onNotebookServersResponse(responseEvent) {
+        const response = responseEvent.detail.response;
+        this.loading = true;
+
+        try {
+            const defaultnotebook = response.notebooks.find((nb)=>
+                nb.labels['notebook.statcan.gc.ca/default-notebook'] &&
+                nb.status.phase === 'stopped'
```

Comment:
Could the status be something else than stopped but still needed to activate? More asking than anything else

Reply:
I don't think so. Based on the statuses we use (https://github.com/StatCan/jupyter-apis/blob/main/frontend/common/kubeflow-common-lib/projects/kubeflow/src/lib/resource-table/status/types.ts), i feel like the only time we would like to attempt to start a notebook is when it is currently stopped.